BUILDING.md

```diff
@@ -94,20 +96,45 @@ Then, run the system preparation scripts and try to compile the project:
 cp components/hab/install.sh /tmp/
 sh support/linux/install_dev_0_ubuntu_latest.sh
 sh support/linux/install_dev_9_linux.sh
-. ~/.profile
+```
+
+When you run the install_dev_9_linux.sh script, you may see this error.
```

Comment:
Would it make sense to merge this in for now, then open an issue saying that the script needs to be fixed?  While I would be happy to take a look at fixing the script, with ChefConf coming up so soon I think my time would be better spent on demo work at least until after ChefConf.  This would also be a great first contribution for a new contributor.